CHANGES.md

```diff
@@ -4,7 +4,7 @@
 ### Notable changes
 
 * Updated REX-Ray version to 0.11.2 (DCOS_OSS-3597) [rexray v0.11.2](https://github.com/rexray/rexray/releases/tag/v0.11.2)
-* Updated to [DC/OS UI 1.11+v1.15.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.15.0/CHANGELOG.md)
+* Updated to [DC/OS UI 1.11+v1.16.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.16.0/CHANGELOG.md)
```

Comment:
IMHO we should add a new line here instead of replacing the old one 🤔

Reply:
Why?

Reply:
when we update ui multiple times, we need to keep track of all the updates 🤷‍♂️

Reply:
🎨  I'd change the line to `Updated DC/OS UI to [1.11+v1.16.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.16.0/CHANGELOG.md)`.